components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -643,165 +596,20 @@ subroutine ma_convproc_dp_intr(                &
    end do
 
 
-! set itmpveca=1 for the 2 "strong convection" grid points, -1 otherwise
-! set itmpveca=associated index to the gathered arrays
    call get_lat_all_p(  lchnk, pcols, lat_ndx )
    call get_lon_all_p(  lchnk, pcols, lon_ndx )
    call get_rlat_all_p( lchnk, pcols, lat_deg )
    call get_rlon_all_p( lchnk, pcols, lon_deg )
    lat_deg(1:ncol) = lat_deg(1:ncol) *180.0/3.1415926536
    lon_deg(1:ncol) = lon_deg(1:ncol) *180.0/3.1415926536
```

Comment:
I think this block of getting lat/lons can also be removed. It was there for printing diagnostics as far as I remember. Would you please verify and remove it if it is not used?